From bf348c9f38358881174888245dea4c1356dad199 Mon Sep 17 00:00:00 2001
From: Abhishek <apatgaonkar@habana.ai>
Date: Tue, 22 Sep 2020 20:17:26 +0300
Subject: [PATCH] [SW-0] goya2: change vuart offsets

During Goya2 PO, existing vuart offsets are not
matching with hl-smi. This patch changes vuart
data and status offsets as per arch team suggestion.

Change-Id: I5360fe27f59dbbad058f3af53bf35530e58a5d35
---

diff --git a/goya2/common.c b/goya2/common.c
index 7030d77..9f698dc 100644
--- a/goya2/common.c
+++ b/goya2/common.c
@@ -201,8 +201,8 @@
 
 void preboot_delay(void) {}
 
-#define mmPCIE_WRAP_VUART_TX_DATA	0xC01F60
-#define mmPCIE_WRAP_VUART_TX_STATUS	0xC01F64
+#define mmPCIE_WRAP_VUART_TX_DATA	0xC14F60
+#define mmPCIE_WRAP_VUART_TX_STATUS	0xC14F64
 void preboot_putc_virt_uart(const char c)
 {
 	/*
